recipes/pyobjcryst/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyobjcryst" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "b48b1a66973dc42d8d15c5ddcf4989cb4160cab220c3a3c60664af46926092d4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - SConstruct.patch
+
+build:
+  number: 0
+  skip: True  # [win or py2k or osx]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.11.2
+    - libobjcryst 2015.1.*  # [py2k]
+    - boost-all 1.59.0      # [py2k]
+    - libobjcryst 2017.1.*  # [py3k]
+    - boost 1.61.0          # [py3k]
+    - scons
+    - toolchain
+  run:
+    - python
+    - setuptools
+    - numpy >=1.11.2
+    - libobjcryst 2015.1.*  # [py2k]
+    - boost-all 1.59.0      # [py2k]
+    - libobjcryst 2017.1.*  # [py3k]
+    - boost 1.61.0          # [py3k]
+
+test:
+  imports:
+    - pyobjcryst
+    - pyobjcryst.tests
+
+about:
+  home: https://github.com/diffpy/pyobjcryst
+  license: BNL
+  license_family: BSD
```

Comment:
Is this actually considered a BSD-family license? Looks kind of different on a skim.

Reply:
Attn: @pavoljuhas

Reply:
The license text is from Intellectual Property office at the BNL and contains licenses of the previous projects we build on.  The 3-clause BSD is sprinkled in the text at

https://github.com/diffpy/pyobjcryst/blob/35ec0dbd47c24ecb43c1fb8b16d256cb3f805ccf/LICENSE.txt#L51-L68
and
https://github.com/diffpy/pyobjcryst/blob/35ec0dbd47c24ecb43c1fb8b16d256cb3f805ccf/LICENSE.txt#L79-L98